selenium.js

```diff
@@ -269,35 +268,17 @@ const run = async (browser, version, os, ctx, task) => {
       throw new Error('Results failed to upload');
     }
 
-    try {
-      log(task, 'Attempting to download results...');
-      if (browser === 'chrome' || browser === 'firefox' ||
-        (browser === 'edge' && version >= 79)) {
-        await goToPage(driver, browser, version, `view-source:${host}/api/results`);
-      } else {
-        await goToPage(driver, browser, version, `${host}/api/results`);
-      }
-      const reportBody = await driver.wait(until.elementLocated(By.css('body')), 10000);
-      const reportString = await reportBody.getAttribute('textContent');
-      log(task, 'Saving results...');
-
-      if (!ctx.testenv) {
-        const report = JSON.parse(reportString);
-        const {filename} = getReportMeta(report);
-        await fs.writeFile(path.join(resultsDir, filename), stringify(report));
-      }
-    } catch (e) {
-      // If we can't download the results, fallback to GitHub
-      log(task, 'Uploading results to GitHub...');
-      await goToPage(driver, browser, version, `${host}/export${ctx.testenv ? '?mock=true' : ''}`);
-      statusEl = await driver.findElement(By.id('status'));
-      await driver.wait(until.elementTextContains(statusEl, 'to'));
-
-      if ((await statusEl.getText()).search('Failed') !== -1) {
-        throw new Error('Pull request failed to submit');
-      }
-
-      // warn('Exported to GitHub');
+    log(task, 'Exporting results...');
+    await goToPage(driver, browser, version, `${host}/export`);
+    // This depends on the <a download> initially having no href attribute.
+    const downloadEl = await driver.wait(until.elementLocated(By.css('#download-url[href]')), 10000);
```

Comment:
This doesn't work on IE6 or 7, I'm guessing the attribute selector doesn't work there.

Reply:
Fixed by https://github.com/foolip/mdn-bcd-collector/pull/1001.